src/Modules/complete_task.js

```diff
@@ -0,0 +1,20 @@
+export const deleteComplete = (tasks) => {
+  const newTasks = tasks.filter((tasks) => tasks.completed === false);
+  tasks = newTasks;
+  for (let i = 0; i < tasks.length; i += 1) {
+    tasks[i].index = i + 1;
+  }
+  return tasks;
+};
+
+const markComplete = (tasks, index, save, load) => {
+  if (tasks[index].completed === false) {
+    tasks[index].completed = true;
+  } else {
+    tasks[index].completed = false;
+  }
```

Comment:
Hey, your logic is very well thought out however this code could be rewritten in a cleaner way, in one line of code, something like this:  `tasks[index].completed = !tasks[index].completed`

Reply:
@NitBravoA92 Thanks for the new perspective about that part of code🙌